io.catenax.serial_part/3.0.0/SerialPart.ttl

```diff
@@ -0,0 +1,206 @@
+#######################################################################
+# Copyright(c) 2024 BASF SE
+# Copyright(c) 2024 Bayerische Motoren Werke Aktiengesellschaft(BMW AG)
+# Copyright(c) 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V.(represented by Fraunhofer ISST & Fraunhofer IML)
+# Copyright(c) 2024 German Edge Cloud GmbH & Co. KG
+# Copyright(c) 2024 Henkel AG & Co. KGaA
+# Copyright(c) 2024 Mercedes Benz AG
+# Copyright(c) 2024 Robert Bosch Manufacturing Solutions GmbH
+# Copyright(c) 2024 SAP SE
+# Copyright(c) 2024 Siemens AG
+# Copyright(c) 2024 T-Systems International GmbH
+# Copyright(c) 2024 ZF Friedrichshafen AG
+# Copyright(c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International(CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:samm:io.catenax.serial_part:3.0.0#>.
+@prefix ext-built: <urn:samm:io.catenax.shared.part_site_information_as_built:1.0.0#>.
```

Comment:
please wait of the new aspect model, as this is missing a wrong link

Reply:
@catroest Do you mean the aspect model PartSiteInformationAsBuilt 1.0.0? It lready exists -> #453.

Reply:
+ there is an update in BPN (#571 ) which then will be also included in #581

Reply:
@catroest Thanks for the important hint! I'll open an new Issue and PR for the adaptations of partSiteInformationAsbuilt. The changes needs to be done for these aspect models as well: 

- #562
- #563
- #567

Reply:
@catroest Thanks! :-)

Reply:
It was merged, now you can use PartSiteInformation 2.0.0